apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -220,11 +248,22 @@ static int make_long_file(void)
 static void tc_fs_vfs_mount(void)
 {
 	int ret;
+#ifdef CONFIG_TASH
+	char *buff = "mount";
+#endif
 	ret = mount(MOUNT_DEV_DIR, CONFIG_MOUNT_POINT, TARGET_FS_NAME, 0, NULL);
 	TC_ASSERT_EQ("mount", ret, OK);
+	/*For each mountpt operation*/
+#if CONFIG_TASH
+		ret = tash_execute_cmd (&buff, 1);
```

Comment:
The tash_execute_cmd function is a TASH internal function. How do you call it?

Reply:
Okay , will remove that code.